combat-trainer.lic

```diff
@@ -326,6 +326,8 @@ class LootProcess
       end
     end
     pause 0.25
+    bput("drop #{item}", 'You drop') if reget(3, 'There isn\'t any more room')
```

Comment:
Is 3 lines of lookback enough?

Would using a Flag (like we do for pouches) be more reliable?

Reply:
2 is too little, 4 is too much, 3 felt just right. We could do a flag too.